### PR TITLE
Slack user extraction scheduler

### PR DESCRIPTION
**Pull request by venkuppu-chn**
### Changes
 - [ ] Why is this change required? What problem does it solve?
 - [ ] If proposing a new change then please raise an issue first

### How Has This Been Tested? (Optional)
 - [ ] Please describe in detail how you tested your changes.
 - [ ] Include details of your testing environment, and the tests you ran to
 - [ ] How your change affects other areas of the code, etc.

### Checklist
 - [ ] tested locally
 - [ ] added new dependencies
 - [ ] updated the docs
 - [ ] added a test

### CODE REVIEW

**Comment by venkuppu-chn** [comment]
@mukul-seagate11 and @shailesh-vaidya 
As part this PR, I would like add below mentioned token and their web cookies in Secrets to run the slack workflow action. Let me know if any concern. 
CEPH
CEPH_COOKIE
DAOS
DAOS_COOKIE
MINIO
MINIO_COOKIE
OPENIO
OPENIO_COOKIE
OPENSTACK
OPENSTACK_COOKIE

File: .github/workflows/slack.yml [hunk 1]
```diff
@@ -29,28 +29,38 @@ jobs:
...
-          pip3 install python-dotenv
+          # pip3 install python-dotenv
             
       # Runs a script using the runners shell
       - name: Run a user extraction script
         working-directory: ./metrics/slack          
         env:
           SLACK_KEY: ${{ secrets.SLACK_KEY }}
-          
+          MINIO: ${{ secrets.MINIO }}
```
Comment (shailesh-vaidya) [inline .github/workflows/slack.yml:39]: @venkuppu-chn  Not sure if we should use the product name in our repository. We have cleaned-up such words as part of contentious word removal activity. 
@johnbent  Could you please suggest.

**Review by shailesh-vaidya — COMMENTED** [review]
Please fix DCO and Codacy check errors.

File: .github/workflows/slack.yml (hunk 1 above)
Comment (venkuppu-chn) [reply .github/workflows/slack.yml:38]: Manually signed-off the DCO this time. I will ensure the same from next time in commit message.

**Comment by venkuppu-chn** [comment]
Closing this PR. Will raise the another.

**Comment by venkuppu-chn** [comment]
Created the new PR, not able resolve the DCO check. No longer required. 

https://github.com/Seagate/cortx/pull/784

**Pull request closed without merging**